schema/me/bidders.js

```diff
@@ -15,8 +15,7 @@ export default {
     },
   },
   resolve: (root, options, { rootValue: { accessToken } }) => {
-    const url = options.sale_id === undefined ? 'me/bidders' : 'bidders';
     if (!accessToken) return null;
-    return gravity.with(accessToken)(url, options);
+    return gravity.with(accessToken)('me/bidders', options);
```

Comment:
Previous behavior would query the `/bidders` endpoint when a `sale_id` was passed, which means that within a `me {}` query, it was returning _all_ bidders for a sale.

You actually can pass a `sale_id` to the `me/bidders` endpoint, which is what should be happening here. 

Also forgive me if there was a reason for the original approach, but given that you're querying for a user's bidders, it didn't make sense to me to return all bidders for a sale... especially since you can't even narrow them down by `user_id`.